lib/pages/scout/bloc/ScoutPageBloc.dart

```diff
@@ -1,37 +1,55 @@
 import 'package:ax_dapp/pages/scout/models/AthleteScoutModel.dart';
 import 'package:ax_dapp/pages/scout/models/ScoutPageState.dart';
 import 'package:ax_dapp/pages/scout/usecases/GetScoutAthletesDataUseCase.dart';
+import 'package:ax_dapp/service/Controller/usecases/GetChainChangesUseCase.dart';
 import 'package:ax_dapp/util/BlocStatus.dart';
 import 'package:ax_dapp/util/SupportedSports.dart';
+import 'package:ax_dapp/util/CurrentChain.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
 import '../models/ScoutPageEvent.dart';
 
 class ScoutPageBloc extends Bloc<ScoutPageEvent, ScoutPageState> {
   final GetScoutAthletesDataUseCase repo;
+  final GetChainChangesUseCase chainChangeUseCase;
+  late Stream _chainChangeStream;
 
-  ScoutPageBloc({required this.repo}) : super(ScoutPageState.initial()) {
+  ScoutPageBloc({required this.repo, required this.chainChangeUseCase})
+      : super(ScoutPageState.initial()) {
     on<OnPageRefresh>(_mapRefreshEventToState);
     on<SelectSport>(_mapSelectSportToState);
     on<OnAthleteSearch>(_mapSearchAthleteEventToState);
+    _chainChangeStream = chainChangeUseCase.stream;

```

Comment:
this is unnecessary

Reply:
This references the getter method for obtaining broadcast stream inside usecase class.